.travis.yml

```diff
@@ -1,47 +1,36 @@
 language: cpp
-sudo: false
+env:
+  global:
+    - GAPROOT=gaproot
+    - COVDIR=coverage
+    - GAP_PKGS_TO_BUILD="io" # exclude profiling, i.e., *this* package
+    - NO_COVERAGE=1  # tests are not compatible with regular coverage collection
```

Comment:
Had to disable coverage computation again as the test suite sadly does not like that, it expects to start with profiling disabled. While that is understandable, it is also a bit sad, as the coverage results would of course be useful. I guess one could solve this with some cleverness, but that's beyond the scope of this PR.

Reply:
It's hard to imagine how to easily test profiling profiling :) could write a very different test suite which first generates some profiles, then test reading and outputting them (which could be profiled).